aws_lambda_powertools/event_handler/openapi/params.py

```diff
@@ -1122,3 +1129,9 @@ def _create_model_field(
         required=field_info.default in (Required, Undefined),
         field_info=field_info,
     )
+
+
+# Public type aliases for form and file parameters
+# Use Annotated types to work properly with Pydantic
+File = Annotated[bytes, _File()]
+Form = Annotated[str, _Form()]
```

Comment:
I was reviewing this code and the reason you created this alias is because customers can annotate File and Form directly in their function?

Something like this:

```py
@app.post("/public-api")
def upload_with_public_types(
    file: File,  # Using the public export
    name: Form,  # Using the public export
):
    return {"status": "uploaded"}
```

I don't if we should allow this because we tell customers to use Annotated types in Query, Path, Header and others. I'd prefer to keep the same pattern and remove this alias and rename original names to `File` and `Form`.